nav2_util/src/node_utils.cpp

```diff
@@ -67,9 +67,9 @@ rclcpp::Node::SharedPtr generate_internal_node(const std::string & prefix)
 {
   rclcpp::NodeOptions options;
   options.use_global_arguments(false);
-  options.start_parameter_services(false);
-  options.start_parameter_event_publisher(false);
-  return rclcpp::Node::make_shared(generate_internal_node_name(prefix));
+  // options.start_parameter_services(false);
```

Comment:
Prefer removing commented-out code. I'm generally not a fan of leaving commented-out code around for documentation purposes, but would instead rather have it in the revision history and keep the current code as clean as possible.

Reply:
Done